README.md

```diff
@@ -11,6 +11,8 @@ The plugin provides tasks for uploading, downloading, running and managing [Goog
 
 To use the App Engine plugin, include in your build script:
 
```

Comment:
actually since we're already in here can you use the ``` format on this?

```
- ```groovy
- apply plugin: 'java'
- apply plugin: 'war' // or 'ear'
- apply plugin: 'appengine'
- ```
```

Reply:
the prevailing other docs seem to say `gradle` instead of `groovy`, so I am guessing that's what's desired here since this is more specifically gradle code.